docs/user-guide/cli.md

```diff
@@ -116,6 +116,10 @@ Produce a report of the `stylelint-disable` comments that used for rules that do
 
 Produce a report to clean up your codebase, keeping only the `stylelint-disable` comments that serve a purpose. [More info](options.md#reportneedlessdisables).
 
+### `--report-unscoped-disables, --rud`
+
+Produce a report of the configuration comments that are not scoped to one or more rules.
```

Comment:
[suggestion] How about adding a "More info" link like other `--report-*` flags?

```suggestion
Produce a report of the configuration comments that are not scoped to one or more rules. [More info](options.md#reportunscopeddisables).
```

Reply:
I thought about it but there's nothing more there.
If you have something to add to _options.md#reportunscopeddisables_ that would warrant the _More info_ link, I would do it.
I could add a _See also_ link but _More info_ is ungenuine.

Reply:
I think we still need the "More info" link to associate the CLI flag with the Node.js API option. Also, for consistency, we should have the "More info" link like other flags have it.